data/mods/ssb/abilities.ts

```diff
@@ -367,7 +379,22 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		name: "I Did It Again",
 		// implemented in rulesets.ts
 	},
-
+	
+	// Rumia
+	youkaiofthedusk: {
```

Comment:
this prankster part will need a dark-type immunity check, will add this when I get home